airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1158,6 +1198,26 @@ components:
     CustomerId:
       type: string
       format: uuid
+    WorkspaceCreate:
+      type: object
+      required:
+        - name
+      properties:
+        email:
+          type: string
+          format: email
+        initialSetupComplete:
+          type: boolean
+        displaySetupWizard:
+          type: boolean
```

Comment:
i think instead of exposing these in the API, you can hide them and just set them to `false` in the handler. i thin it's reasonable that these 2 fields will always be false when the workspace is created.